content/integrations/azure.md

```diff
@@ -233,13 +244,28 @@ Naviate to the [Azure VM Default Dashboard][6] to see this dashboard populate wi
 
 Learn more about how to monitor Azure VM performance metrics with [our series of posts](https://www.datadoghq.com/blog/how-to-monitor-microsoft-azure-vms/). We detail the key performance metrics, how to collect them, and how to use Datadog to monitor Azure VMs.
 
+### Configuration
+{: #configuration}
+
+Optionally, you can limit the Azure VMs that are pulled into Datadog by entering tags in the "Optionally filter to VMs with tag" textbox. This comma separated list of tags (in the form 'key:value') defines a filter that we will use when collecting metrics from Azure VMs. Wildcards, such as '?' (for single characters) and '*' (for multiple characters) can also be used. Only VMs that match one of the defined tags will be imported into Datadog. The rest will be ignored.
+
+VMs matching a given tag can also be excluded by adding '!' before the tag. 
+
+~~~
+e.x. datadog:monitored,env:production,!env:staging,instance-type:c1.*
```

Comment:
`e.g.`, not `e.x.`. Also `.e.g.` or `For example:` should appear just before the code block, not within code block.